scenarios/typescript/node/captioning/caption.ts

```diff
@@ -0,0 +1,313 @@
+"use strict";
+
+/*
+Dependencies
+*/
+
+/* Notes:
+- To install dependencies for this sample:
+npm install microsoft-cognitiveservices-speech-sdk
+- The Speech SDK on Windows requires the Microsoft Visual C++ Redistributable for Visual Studio 2019 on the system. See:
+https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-sdk?tabs=windows%2Cubuntu%2Cios-xcode%2Cmac-xcode%2Candroid-studio
+*/
+import * as sdk from "microsoft-cognitiveservices-speech-sdk";
```

Comment:
@v-jaswel - Just a general comment on the JavaScript sample I don't see any dot TS examples in carbon. All of our samples that I found are dot JS.